client/errors.go

```diff
@@ -18,6 +18,8 @@ func unwrapErr(err error) error {
 
 // IsErrContainerNotFound checks if err corresponds to NeoFS status
 // return corresponding to missing container. Supports wrapped errors.
+//
+// Deprecated: use errors.Is(err, apistatus.ErrContainerNotFound) instead.
```

Comment:
just purge all of them

Reply:
We can keep them for a release, it'd be a bit easier to migrate for people.

Reply:
we can force migration now, it's better than upsetting someone after release

Reply:
How I remember @roman-khimov also suggested to remove these functions, at the beginning. Deprecation was an alternative way, let's drop them now.

Reply:
OK

Reply:
Removed